cmd/buildah/common.go

```diff
@@ -108,3 +111,50 @@ func getDateAndDigestAndSize(image storage.Image, store storage.Store) (time.Tim
 	}
 	return created, manifestDigest, imgSize, err
 }
+
+// ContextFromGlobalOptions returns a SystemContext populated with values
+// per the input parameters provided by the caller for the use in authentication.
+func ContextFromGlobalOptions(c *cli.Context, flagPrefix string) (*types.SystemContext, error) {
```

Comment:
We're not currently using `flagPrefix`.  Are there plans to use it?  If not, can we drop it?